File: testers/grpc/java-quarkus/generated-and-checked-in/com/example/grpc/OrderId.java
package com.example.grpc;

/** Wrapper type for typr.grpc.GrpcCodegen$$$Lambda/0x00007fc001118400@41330d4f */
public record OrderId(String value) {
  public OrderId withValue(String value) {
    return new OrderId(value);
  }

  @Override
  public java.lang.String toString() {
    return value.toString();
  }

  /** Create a OrderId from a raw value */
  public static OrderId valueOf(String v) {
    return new OrderId(v);
  }

  /** Get the underlying value */
  public String unwrap() {
    return this.value();
  }
}

File: testers/grpc/scala-cats/generated-and-checked-in/com/example/grpc/OrderId.scala
package com.example.grpc



/** Wrapper type for typr.grpc.GrpcCodegen$$$Lambda/0x00001e00012fb730@52336435 */
case class OrderId(value: String) extends scala.AnyVal {
  /** Get the underlying value */
  def unwrap: String = {
    return this.value
  }
}

object OrderId {
  /** Create a OrderId from a raw value */
  def valueOf(v: String): OrderId = {
    return new OrderId(v)
  }
}